backend/contactsAPI.py

```diff
@@ -39,6 +43,28 @@ def get_contacts():
         return Response(status=204)
 
 
+@app.route('/img', methods=['POST'])
+def image():
+    if request.method == 'POST':
+        token = request.headers.get('token')
+        _id = request.form.get('_id')
+        user = User().find_by_token(token)
+        image = request.files['file']
+        hashedName = image.split(".", 1)[0] + str(hash(_id)) + str(hash(datetime.now().strftime("%d")))
+        contact = Contacts().find_by_id(_id)
```

Comment:
```python
contact = Contacts(request.get_json()['contact'])                 
contact.save()
```
This will either create a new contact, or update the current contact.

Reply:
That way you can avoid finding by _id.

Reply:
@ysunkara Nevermind, the API is different than I expected here, you can ignore these comments.